api/controllers/ApplicationController.js

```diff
@@ -216,36 +216,40 @@ var applicationController = {
      */
     submitApplication: function(req, res) {
         var id = req.query.merchantReturnData;
-        console.log(id + " - attempting to submit application");
+        sails.log.info(id + " - attempting to submit application");
         Application.findOne({
             where: {
                 application_id: id
             }
         }).then(function(application) {
             if (application !== null) {
-              console.log(id + " - has returned from barclays");
+              sails.log.info(id + " - has returned from barclays");
               if (application.submitted === 'draft') {
-                console.log(id + " - has not been submitted previously");
-                console.log(id + " - exporting app data");
-                applicationController.exportAppData(req, res);
+                sails.log.info(id + " - has not been submitted previously");
+                sails.log.info(id + " - exporting app data");
+                applicationController.exportAppData(req, application);
               } else {
-                console.log(id + " - has been submitted previously");
+                sails.log.info(id + " - has been submitted previously");
               }
               if (application.serviceType == 1) {
-                console.log(id + " - displaying standard confirmation page to user");
+                sails.log.info(id + " - displaying standard confirmation page to user");
                 return applicationController.confirmation(req, res);
-              }
-              else {
+              } else if (application.serviceType === 4) {
+                sails.log.info(
+                    id + ' - displaying eApostille confirmation page to user'
+                );
+                return res.view('eApostilles/applicationSubmissionSuccessful.ejs',{});
+              } else {
                 var businessApplicationController = require('./BusinessApplicationController');
-                console.log(id + " - displaying business confirmation page to user");
+                sails.log.info(id + " - displaying business confirmation page to user");
                 return businessApplicationController.confirmation(req, res);
               }
             } else {
-                console.log(id + " - could not be found in db");
+                sails.log.info(id + " - could not be found in db");
             }
         }).catch(function(error) {
-          console.log(id + " - has encountered an error");
-          console.log(id + error);
+          sails.log.info(id + " - has encountered an error");
+          sails.log.info(id + error);
```

Comment:
this should be something like `sails.log.error(id, "has encountered an error", error)`. 

What usually happens in production is that these logs are filtered by level. So, in dev envs or test envs you might want to see more logs, but maybe not "debug" logs. You don't want ALL the logs in production because they have to be stored somewhere - so filling them up with lower level logs (when you know it's probably fine) can get costly (and clogs up the logs so it's difficult to see the errors!)